frontend/src/views/Leaderboard.vue

```diff
@@ -48,7 +48,7 @@ export default {
 
   async created() {
     this.waitingLeaderboardOutcome = true;
-    const leaderboardData = await fetch(apiUrl('static/leaderboard'));
+    const leaderboardData = await fetch(apiUrl('static/leaderboard/' + localStorage.getItem('currentChain')));
```

Comment:
What happens if there is nothing in localStorage? Maybe do a fallback?
`localStorage.getItem('currentChain') || 'BSC';`